src/millie.nim

```diff
@@ -10,46 +10,41 @@ const units = @[
   "P", # quadrillion
 ]
 
-## Converts big numbers to what's pleasant to see (an adorable, little girl, perhaps?)
+
 proc millify*(
     value: int64,
     precision: int = 1,
     lowercase: bool = false,
     space: bool = false,
   ): string =
+  ## Converts big numbers to what's pleasant to see (an adorable, little girl, perhaps?)
```

Comment:
In Nim comments for procedures must be inside them, not before

Reply:
Yeah, thanks. Is it a standard convention?

Reply:
@bichanna it's more than a convention - documentation comments (the ones with two ##) are a part of AST and the documentation generator (`nim doc`) will only understand that the comments belong to the proc if you put them like this